### DISCUSSION TITLE
Discuss what should happen when a player finishes a level or song

### DISCUSSION THREAD

**Discussion by mxd1760** [mxd1760/song-typing#27] [Ideas]

**Comment by mxd1760** [comment]
I think that players finishing the text should have some results though whether that's at the end of the song or the end of the text would probably be up to the user to decide with an option somewhere. there's also the use case of users just doing typing practice where they may have a lot of text or want to do a stamina mode type thing going from one file to another.

Another consideration is whether a results screen displays when the user quits from the pause menu or if there should be a separate button for that.

**Comment by mxd1760** [comment]
For now, let's just make something simple and have it based on the user completing typing. We can then keep our minds open for other solutions later on if it feels bad.

**Comment by Jayman2000** [comment]
We could use the [LRC A2 extension](https://en.wikipedia.org/wiki/LRC_(file_format)#A2_extension_(Enhanced_LRC_format)). We could convert the LRC file into an animation track, and then play it with the AnimationPlayer. Then, we could use AnimationPlayer’s built-in interpolation functionality.

Edit: Oops wrong thread. This was supposed to go here: <https://github.com/mxd1760/song-typing/discussions/29>
